ci/scripts/utils/launch_java_agent.sh

```diff
@@ -56,22 +126,61 @@ cd "${JENKINS_AGENT_LANUCH_DIR}"
 
 if ! [[ -f agent.jar ]]; then
   curl -sO "${controller_url}/jnlpJars/agent.jar"
+  echo "Updated agent.jar downloaded"
+fi
+
+if [[ ! -f "${controller_user_auth_token}" ]]; then
+   echo "User Jenkins authetication TOKEN to the controller for using the Remote API does not exist"
+   exit 1
 fi
+JENKINS_TOKEN=$(cat "${controller_user_auth_token}")
+
+echo -e "#!/usr/bin/env python
+import json,sys
+with open(sys.argv[1], 'r') as file:
+    data = json.load(file)
+print(data.get('offline','True'))
+" > parse.py
```

Comment:
Using a heredoc would be a more standard way to do all these multiline strings.
```suggestion
cat << EOF > parse.py
#!/usr/bin/env python3
import json,sys
with open(sys.argv[1], 'r') as file:
    data = json.load(file)
print(data.get('offline','True'))
EOF
```